src/class-scripts.php

```diff
@@ -124,6 +124,11 @@ public function enqueue_js_tracker(): void {
 			$js_api_key = "window.wpParselyApiKey = '" . esc_js( $this->parsely->get_api_key() ) . "';";
 			wp_add_inline_script( 'wp-parsely-loader', $js_api_key, 'before' );
 		}
+
+		if ( isset( $parsely_options['disable_autotrack'] ) && $parsely_options['disable_autotrack'] ) {
```

Comment:
Would it be better to have something like `&& true === && $parsely_options['disable_autotrack']` here? (haven't checked whether we need a boolean or string).